### PR TITLE
Bugfix in oasis.py if lresume is ...

### PR DESCRIPTION
**Pull request by nwieters**
I fixed the wrong `if lresume is` ... statements to `if lresume ==` ...

### CODE REVIEW

File: src/esm_runscripts/oasis.py [hunk 1]
```diff
@@ -100,12 +100,12 @@ def add_coupling(
             sep = ":"
 
         if export_mode == "DEFAULT":
-            if lresume is False:
+            if lresume == False:
```
Comment (pgierz) [inline src/esm_runscripts/oasis.py]:
```suggestion
            if bool(lresume) is False:
```

File: src/esm_runscripts/oasis.py [hunk 2]
```diff
@@ -100,12 +100,12 @@ def add_coupling(
...
                 export_mode = "EXPOUT"
             else:
                 export_mode = "EXPORTED"
 
-        if lresume is False:
+        if lresume == False:
```
Comment (pgierz) [inline src/esm_runscripts/oasis.py]:
```suggestion
        if bool(lresume) is False:
```

**Review by pgierz — CHANGES_REQUESTED** [review]
Small changes to ensure that `lresume` is transformed into a bool at checking time

**Review by mandresm — COMMENTED** [review]
#bump!

**Pull request merged**